src/containers/BooksForm.js

```diff
@@ -14,6 +18,17 @@ class BooksForm extends Component {
     });
   };
```

Comment:
Let's make use of destructuring in the `handleChange` method.
We can rewrite is as follows:
```ruby
`handleChange({ target: { id, value } })`=> {
    this.setState({
      id: value,
    });
  };
```

Reply:
Thanks. changes applied!